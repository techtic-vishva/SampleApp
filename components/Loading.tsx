import { ActivityIndicator } from 'react-native'

export default function Loading() {
	return <ActivityIndicator style={{ marginTop: 'auto', marginBottom: 'auto' }} />
}
