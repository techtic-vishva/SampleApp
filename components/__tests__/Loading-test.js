import * as React from 'react'
import renderer from 'react-test-renderer'

import Loading from '../Loading'

it('Loading renders correctly', () => {
	const tree = renderer.create(<Loading />).toJSON()
	expect(tree).toMatchSnapshot()
})
