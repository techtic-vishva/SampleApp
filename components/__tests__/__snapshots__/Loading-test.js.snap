// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correctly 1`] = `
<ActivityIndicator
  style={
    Object {
      "marginBottom": "auto",
      "marginTop": "auto",
    }
  }
/>
`;
